rskj-core/src/test/java/co/rsk/rpc/modules/eth/EthModuleTest.java

```diff
@@ -353,4 +355,359 @@ void chainId() {
         );
         assertThat(eth.chainId(), is("0x21"));
     }
+
+    @Test
+    void whenExecuteCallWithDataParameter_callExecutorWithData() {
```

Comment:
nice coverage 👍 could we also test `CallArguments` as an unit? Eg as in here? - `web3_CallArguments_toString()`

Reply:
Done